packages/ihe-gateway/server/CodeTemplates/Common Library/Set SAML Attributes/Set SAML Attributes.js

```diff
@@ -5,35 +5,52 @@
 	@param {Object} msg - original msg object
 */
 function setSAMLAttributes(msg) {
+  // The name of the organization that the user belongs to as required by HIPAA Privacy Disclosure Accounting
+  channelMap.put("Organization", Config.getHomeCommunityName());
```

Comment:
Another inconsistency in the text casing for `channelMap.put`.
We use all caps in other places..

Reply:
I only replicated what the code already had. If you look at the previous version of this file - and in multiple places in the code base, it's referring to `channelMap`, not `ChannelMap`. In fact, I searched the code on `develop` and I get 196 hits, vs. 0 with PascalCase.

Maybe you mixed them up?

Reply:
If you're referring to the value, that's what it was before, so I'd rather not change it now w/o proper testing.